CHANGELOG.md

```diff
@@ -30,6 +218,7 @@ Generated on 2025-04-17
 ### Bugs Fixed
 |||
 |:---|:---|
+|[#11235](https://github.com/NVIDIA/spark-rapids/issues/11235)|[BUG] 229 Unit tests failed in Spark 400 against cuda 11 and scala 2.13|
```

Comment:
This is fixed in 25.06 with PR https://github.com/NVIDIA/spark-rapids/pull/12813. May update the issue's project and regenerate the change log.

Reply:
Updated the issue's roadmap. And regenerating.